### DISCUSSION TITLE
Individual Time Parameter for GLTF Nodes

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#261] [Feature Requests]
right now we have a single time input for the whole GLTF scene, controlling all animations at once.
it would be really helpful for an ongoing project to be able to control the time for single GLTF objects, exposed by the GLTF scene op. 

Let's say you have a bird animation and you want that bird three times but be able to control the animation speed of this bird independently. 

![image](https://user-images.githubusercontent.com/1138673/171810085-0e66695f-a56b-42ec-8809-bef6758cf3bd.png)

**Comment by wirmachenbunt** [comment]
which line in GLTF scene is controlling the progress of the animation 
![image](https://user-images.githubusercontent.com/1138673/171879695-dab1ad9d-c1f3-44f7-99d3-b2a132a151d1.png)
this seems not it, can you give me a hint ?

**Comment by pandrr** [comment]
- the skin op has a seperate time input
- others would be handy but not yet implemented
- animation is mainly done in the transform function in the attachment node.js
